### DISCUSSION TITLE
About the Comparison between AID and default data aug in mmpsoe

### DISCUSSION THREAD

**Discussion by WeianMao** [open-mmlab/mmpose#1064] [General] [answered]
Hi,
My model overfit the coco dataset, so i try to apply some data aug to address the overfitting issue.
in the mmpsoe, there are color aug and crop aug. mean while, the anthor also implement the AID for mmpose. have you ever compare the default data aug in mmpose with AID? thank you very much!

**Comment by ly015** [comment]
Could you provide some help here? @jin-s13 @innerlee

**Comment by jin-s13** [comment, marked as answer]
It seems that in AID's paper, the models are first pre-trained on COCO for 210 epochs with the default settings and then fine-tuned with AID for another 210 epochs. The authors report that this training strategy will bring some gains, but we did not fully check this.

**Comment by WeianMao** [comment]
oh, thanks. i will try both. once i have result i will paste it here
